src/components/ha-paper-slider.html

```diff
@@ -0,0 +1,80 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-slider/paper-slider.html">
+
+
+<dom-module id="ha-paper-slider">
+  <template strip-whitespace>
+    <style include="paper-slider">
+      .pin > .slider-knob > .slider-knob-inner {
+        font-size:  var(--ha-paper-slider-pin-font-size, 10px);
+        line-height: normal;
+      }
+
+      .pin > .slider-knob > .slider-knob-inner::before {
+        top: unset;
+        margin-left: unset;
+
+        bottom: calc(15px + var(--calculated-paper-slider-height)/2);
+        left: 50%;
+        width: 2.6em;
+        height: 2.6em;
+
+        -webkit-transform-origin: left bottom;
+        transform-origin: left bottom;
+        -webkit-transform: rotate(-45deg) scale(0) translate(0);
+        transform: rotate(-45deg) scale(0) translate(0);
+      }
+
+      .pin.expand > .slider-knob > .slider-knob-inner::before {
+        -webkit-transform: rotate(-45deg) scale(1) translate(7px, -7px);
+        transform: rotate(-45deg) scale(1) translate(7px, -7px);
+      }
+
+      .pin > .slider-knob > .slider-knob-inner::after {
+        top: unset;
+        font-size: unset;
+
+        bottom: calc(15px + var(--calculated-paper-slider-height)/2);
+        left: 50%;
+        margin-left: -1.3em;
+        width: 2.6em;
+        height: 2.4em;
+
+        -webkit-transform-origin: center bottom;
+        transform-origin: center bottom;
+        -webkit-transform: scale(0) translate(0);
+        transform: scale(0) translate(0);
+      }
+
+      .pin.expand > .slider-knob > .slider-knob-inner::after {
+        -webkit-transform: scale(1) translate(0, -10px);
+        transform: scale(1) translate(0, -10px);
+      }
+    </style>
+  </template>
+
+  <script>
+    customElements.whenDefined('paper-slider').then(() => {
```

Comment:
Why do you need "whenDefined"? There is no async here, as you import paper-slider.

Reply:
See this comment. https://github.com/Polymer/polymer/issues/4556#comment-295968529
But maybe I misunderstood that.

But we don’t need this anyway because we’ll clone the whole element instead of extending it (see balloobs comment above.)

Reply:
Just for the record, according to https://github.com/Polymer/polymer/issues/4556#issuecomment-295969325 whenDefined wasn't needed. (Unless used from the index doc, which AFAIU is not our case)